src/AWS.Deploy.Recipes/RecipeDefinitions/BlazorWasm.recipe

```diff
@@ -8,6 +8,7 @@
     "CdkProjectTemplate": "../CdkTemplates/BlazorWasm",
     "CdkProjectTemplateId": "netdeploy.BlazorWasm",
     "Description": "Deploy a Blazor WebAssembly application to an Amazon S3 bucket. The Amazon S3 bucket created during deployment will be configured for web hosting and its contents will be open to the public with read access.",
+    "ShortDescription": "Deploy a Blazor WebAssembly application to an Amazon S3 bucket.",
```

Comment:
Suggested text:

"Blazor WebAssembly application hosted in an Amazon S3 bucket."

Reply:
Fixed